modules/cms/lang/vn/lang.php

```diff
@@ -0,0 +1,293 @@
+<?php
+
+return [
+    'cms_object' => [
+        'invalid_file' => 'Tên tệp không hợp lệ :name. Tên tệp có thể chứa chỉ các ký tự chữ và số, dấu gạch dưới, dấu gạch ngang và dấu chấm. Ví dụ: page.htm, page, subdirectory/page',
+        'invalid_property' => "Khổng thể cài thuộc tính ':name'",
+        'file_already_exists' => "Tệp tin ':name' đã tồn tại.",
+        'error_saving' => "Lỗi lưu file ':name'. Vui lòng kiểm tra write permission.",
+        'error_creating_directory' => 'Không thể tạo thư mục :name. Vui lòng kiểm tra write permission.',
+        'invalid_file_extension' => 'Extension không hợp lệ: :invalid. Extensions hợp lệ: :allowed.',
+        'error_deleting' => "Không thể xóa template file ':name'. Vui lòng kiểm tra write permission.",
+        'delete_success' => 'Đã xóa template: :count.',
+        'file_name_required' => 'Tên file là bắt buộc.',
+        'safe_mode_enabled' => 'Safe mode đã được bật.'
+    ],
+    'dashboard' => [
+        'active_theme' => [
+            'widget_title_default' => 'Website',
+            'online' => 'Online',
+            'maintenance' => 'Đang bảo trì',
+            'manage_themes' => 'Quản lý themes',
+            'customize_theme' => 'Tùy chỉnh theme'
+        ]
+    ],
+    'theme' => [
+        'not_found_name' => "Theme ':name' không tồn tại",
+        'by_author' => 'Tạo bởi :name',
+        'active' => [
+            'not_set' => 'Theme đang kích hoạt chưa được cài đặt.',
+            'not_found' => 'Không tìm thấy theme đang kích hoạt.'
+        ],
+        'edit' => [
+            'not_set' => 'Theme chưa được cài đặt.',
+            'not_found' => 'Không tìm thấy theme.',
+            'not_match' => "Theme này đang trong chế độ chỉnh sửa. Vui lòng tải lại trang"
+        ],
+        'settings_menu' => 'Giao diện front-end',
+        'settings_menu_description' => 'Quản lý  giao diện front-end và các tùy chỉnh',
+        'default_tab' => 'Các thuộc tính',
+        'name_label' => 'Tên',
+        'name_create_placeholder' => 'Tên theme',
+        'author_label' => 'Tác giả',
+        'author_placeholder' => 'Tên cá nhân hoặc công ty',
+        'description_label' => 'Mô tả',
+        'description_placeholder' => 'Mô tả theme',
+        'homepage_label' => 'Trang chủ',
+        'homepage_placeholder' => 'Đường dẫn trang chủ tác giả',
+        'code_label' => 'Code',
+        'code_placeholder' => 'Tạo mã code duy nhất để phân phối theme của bạn',
+        'preview_image_label' => 'Hình ảnh xem trước theme',
+        'preview_image_placeholder' => 'Đường dẫn hình ảnh xem trước của theme.',
+        'dir_name_label' => 'Directory name',
```

Comment:
There's still a bunch of untranslated strings in here, please remove them. The reason I don't want them in translation files is that the language system already falls back to the default translation (English), so it's redundant to have them repeated and also doesn't propagate the changes to the English translation by default.

Reply:
I've updated some string in that file